client/js/app/stores/ExplorerStore.js

```diff
@@ -283,6 +283,10 @@ function _wrapGroupBy(group_by) {
   return group_by;
 }
 
+function _coerceToNumber(str) {
+  return str && typeof(str) === 'string' ? Number(str) : str;
```

Comment:
Yeah I'd take a look at that in a separate PR.

Good catch on the empty string. Is the empty string *the only* other value this function will be called with? If so, then I'd be more explicit about it and do a check for an empty string instead. Ex: `return str === "" ? str : Number(str)`. And then you likely remove the need for the `_coerceToNumber` entirely, as it's only wrapping a very small piece of logic used in only 2 spots. You can also do `parseFloat(str) || str` as `parseFloat` will return `NaN` for empty strings.